app/models/student.rb

```diff
@@ -0,0 +1,13 @@
+class User < ActiveRecord::Base
+  attr_accessible :title, :first_name, :middle_name, :last_name, :email, :birth_date, :gender
+
+  # allow for null values where applicable to allow for single names
+  # allow for null values where applicable since you cannot legally force a user to identify their gender
+  validates :title, length: { maximum: 10 }
+  validates :last_name, :length: { maximum: 50 } , presence: { message: "Please enter your family name" }
```

Comment:
Oops 1 small syntax issue here I forgot

```suggestion
  validates :last_name, length: { maximum: 50 } , presence: { message: "Please enter your family name" }
```